### PR TITLE
Docs/v24 styling

### PR DESCRIPTION
**Pull request by rolfsmeds**
Styling tabs with CSS selector lists added to Components pages

### CODE REVIEW

**Review by jouni — APPROVED** [review]
Love the `+++<wbr>+++` parts 😄

**Comment by jouni** [comment]
Oh, now I noticed you used the horizontal description list from AsciiDoc. That’s not actually needed, as the default styling in DSP is horizontal, and it automatically wraps to a column on small viewports. But the built-in horizontal style in AsciiDoc renders it as a table, and the DSP styling doesn't apply to that. So I would want to convert these pages to use the default description lists instead.

**Comment by rolfsmeds** [comment]
It does actually wrap them to a column... but I now see that they seem to render identically without the `[Horizontal]` too, so no point in having it there. I'll do a PR.

**Comment by jouni** [comment]
> It does actually wrap them to a column

Oh, really? That’s interesting, I need to try and see for myself. I don't remember implementing any styles for that, so I wonder how the table columns get wrapped…

File: dspublisher/theme/global.css [hunk 1]
```diff
@@ -362,6 +362,20 @@ html[theme~='dark'] .card.card {
...
+  border: none;
+  margin: 0;
+  padding: 0;
+}
+
+/* Make <code> content in <dd>s break at non-word-chars rather than within words */
+[class^='ArticleLayout'] dd code {
+  overflow-wrap: normal;
+  word-break: normal;
+}
```
Comment (jouni) [inline dspublisher/theme/global.css:377]: These styles wouldn't work with the horizontal description lists, as those get rendered inside table elements.

**Comment by rolfsmeds** [comment]
Hmm, maybe it's the horizontal tag that doesn't have any effect for me locally? Because they seem to render 100% identically for me with and without that tag.

Anyhoo, doesn't matter, already merged removal of the tags.

**Pull request merged**